src/GafferScene/LightTweaks.cpp

```diff
@@ -256,6 +256,28 @@ void numericTweak( Data *data, PlugType *plug, LightTweaks::TweakPlug::Mode mode
 	}
 }
 
+template<typename PlugType>
+void splineTweak( Data *data, PlugType *plug, LightTweaks::TweakPlug::Mode mode )
```

Comment:
How do you feel about this? It seems a bit disappointing that we need a special case here - I guess it's highlighting the fact that we are deviating a bit from the other ValuePlugs in our SplinePlug design. The only way I can see that might work is if our new `SplinePlug::ValueType` struct provided an implicit conversion to `Spline` - what do you think to that?